packages/editor-ui/src/Interface.ts

```diff
@@ -67,9 +68,26 @@ declare global {
 			reset?(resetDeviceId?: boolean): void;
 			onFeatureFlags?(callback: (keys: string[], map: FeatureFlags) => void): void;
 			reloadFeatureFlags?(): void;
+			people?: {
+				set(metadata: Record<string, unknown>): void;
+			};
+			feature_flags?: {
+				getFlags(): Record<string, unknown>;
+			};
+			track?(name: string, properties?: ITelemetryTrackProperties): void;
+			register?(metadata: Record<string, unknown>): void;
+			capture?(name: string, properties?: ITelemetryTrackProperties): void;
+		};
+		featureFlag: {
```

Comment:
should be optional. only loaded by posthog file so only get set there.. though not sure how this is used..

Reply:
Removed.